Cargo.lock

```diff
@@ -28,6 +28,12 @@ dependencies = [
  "memchr",
 ]
 
+[[package]]
+name = "android-tzdata"
```

Comment:
I'm really surprised, why do we depend on this? Is there a feature we can disable?

Reply:
Thanks for pointing it out. This came from `serde_with` which we apparently don't need at all.